nss-cohort-36/bangazon-client-virtual-magicians-1
# Cb location filter

# Description
Dropdown with locations that exist in the database. Currently has a pop-up showing the chosen location. Will route to list of products eventually.

Please delete options that are not relevant.

# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout cb-location-filter`
4. go to 'http://localhost:3000/location' and confirm that a dropdown exists with Nashville and Memphis as options. Currently hitting submit only gives a pop-up showing the chosen location.

# Checklist:
